benchmark/Streamly/Benchmark/Data/Array/Stream/Foreign.hs

```diff
@@ -188,18 +188,16 @@ o_1_space_copy_toChunks_group_ungroup env =
 -------------------------------------------------------------------------------
 
 moduleName :: String
-moduleName = "Data.Array.Stream.Foreign"
+moduleName = "Data.Array.Stream.Foreign" 
```

Comment:
Remove trailing space. You can setup your editor to automatically remove trailing spaces on save.